src/components/datasets/DatasetForm.js

```diff
@@ -18,38 +20,53 @@ const validateJson = (rule, value) => {
     }
 };
 
-const DatasetForm = ({ initialValue, formRef }) => {
+const DatasetForm = ({ initialValue, form, updateMode}) => {
+    const discoveryValidator = useDiscoveryValidator();
     return (
-        <Form ref={formRef} layout="vertical">
+        <Form form={form} layout="vertical" initialValues={initialValue}>
             <Item
                 label="Title"
                 name="title"
-                initialValue={initialValue?.title || ""}
+                // initialValue={initialValue?.title || ""}
```

Comment:
does this break editing, or is it handled by the form obj?

Reply:
Removed the commented code. Initial values are handled by the form object using the Item's name, except for the two DropBoxJsonSelect components, which has its own initial value mechanism.